pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -246,6 +263,7 @@ void processArguments() throws Exception {
             } else {
                 this.sinkConfig = new SinkConfig();
             }
+            log.info("The sinkConfig read from file is {}", sinkConfig);
```

Comment:
This logging looks like for debugging purpose. I think it can be removed.

If you want to keep it, it should be moved to after line 262.

Reply:
changed